README.md

```diff
@@ -7,247 +7,37 @@ Signals is a performant state management library with two primary goals:
-
-const counter = signal(0);
-effect(() => console.log(counter.value));
-
-batch(() => {
-	batch(() => {
-		// Signal is invalidated, but update is not flushed because
-		// we're still inside another batch
-		counter.value = 1;
-	});
-
-	// Still not updated...
-});
-// Now the callback completed and we'll trigger the effect.
-```
-
-### `untracked(fn)`
-
-In case when you're receiving a callback that can read some signals, but you don't want to subscribe to them, you can use `untracked` to prevent any subscriptions from happening.
-
-```js
-const counter = signal(0);
-const effectCount = signal(0);
-const fn = () => effectCount.value + 1;
-
-effect(() => {
-	console.log(counter.value);
-
-	// Whenever this effect is triggered, run `fn` that gives new value
-	effectCount.value = untracked(fn);
-});
-```
+- [Core API](./packages/core/README.md#guide--api)
+  - [`signal(initialValue)`](./packages/core/README.md#signalinitialvalue)
+    - [`signal.peek()`](./packages/core/README.md#signalpeek)
+  - [`computed(fn)`](./packages/core/README.md#computedfn)
+  - [`effect(fn)`](./packages/core/README.md#effectfn)
+  - [`batch(fn)`](./packages/core/README.md#batchfn)
+  - [`untracked(fn)`](./packages/core/README.md#untrackedfn)
+- [Preact Integration](./packages/preact/README.md#preact-integration)
+  - [Hooks](./packages/preact/README.md#hooks)
+  - [Rendering optimizations](./packages/preact/README.md#rendering-optimizations)
+    - [Attribute optimization (experimental)](./packages/preact/README.md#attribute-optimization-experimental)
+  - [Utility Components and Hooks](./packages/preact/README.md#utility-components-and-hooks)
+    - [Show Component](./packages/preact/README.md#show-component)
+    - [For Component](./packages/preact/README.md#for-component)
+    - [Additional Hooks](./packages/preact/README.md#additional-hooks)
+      - [`useLiveSignal`](./packages/preact/README.md#uselivesignal)
+      - [`useSignalRef`](./packages/preact/README.md#usesignalref)
+- [React Integration](./packages/react/README.md#react-integration)
+  - [Babel Transform](./packages/react/README.md#babel-transform)
+  - [`useSignals` hook](./packages/react/README.md#usesignals-hook)
+  - [Hooks](./packages/react/README.md#hooks)
+  - [Using signals with React's SSR APIs](./packages/react/README.md#using-signals-with-reacts-ssr-apis)
+  - [Rendering optimizations](./packages/react/README.md#rendering-optimizations)
+  - [Utility Components and Hooks](./packages/react/README.md#utility-components-and-hooks)
+    - [Show Component](./packages/react/README.md#show-component)
+    - [For Component](./packages/react/README.md#for-component)
+    - [Additional Hooks](./packages/react/README.md#additional-hooks)
+      - [`useLiveSignal`](./packages/react/README.md#uselivesignal)
+      - [`useSignalRef`](./packages/react/README.md#usesignalref)
+  - [Limitations](./packages/react/README.md#limitations)
+- [License](#license)
```

Comment:
IMO, really helpful to have this in the repo root for quickly navigating. Lots of folks end up in this repo looking for docs, having links to everything makes it much easier for them to jump to what they're looking for.